scripts/build/builders/builder.py

```diff
@@ -27,10 +27,11 @@ class Builder(ABC):
 
   """
 
-  def __init__(self, root, runner, output_prefix: str ='out'):
+  def __init__(self, root, runner, output_prefix: str = 'out', create_flashbundle: bool = False):
```

Comment:
nit: consider `enable_flashbundle` instead of `create` with the logic of this seemingly being optional (some platforms like telink do not yet have the option)